plugins/logitech-bulkcontroller/fu-logitech-bulkcontroller-device.c

```diff
@@ -328,9 +326,15 @@ fu_logitech_bulkcontroller_device_send_sync_cmd(FuLogitechBulkcontrollerDevice *
 					G_LITTLE_ENDIAN,
 					error))
 		return FALSE;
-
-	verify_buf = (gchar *)&cmd_tmp_64;
-	verify_cmd = g_ascii_strtoll(verify_buf, &pEnd, BASE_10);
+	if (fu_memcpy_safe(ack_payload,
+			   sizeof(ack_payload),
+			   0x0,
+			   (guint8 *)&cmd_tmp_64,
+			   sizeof(cmd_tmp_64),
+			   0x0,
+			   SYNC_ACK_PAYLOAD_LENGTH,
+			   error))
```

Comment:
Nearly. If this fails you need to return FALSE to return the error.

Reply:
Agreed. Fixed